tests/app/billing/test_billing.py

```diff
@@ -339,8 +339,6 @@ def test_get_yearly_billing_usage_summary_from_ft_billing_returns_empty_list_if_
     assert response.get_json() == []
 
 
-# Cannot be ran in parallel - Gathers all
```

Comment:
Was true, it gathers all for the given year, so I gave it a year that had no records.